app/locales/it/locationTracking.json

```diff
@@ -9,4 +9,5 @@
     "latest_news":"Ultime Novità",
     "url_info":"Per più informazioni, visita la homepage di Private Kit:",
     "private_kit_url":"privatekit.mit.edu"
+    "overlap": "CONTROLLA SOVRAPPOSIZIONI"
```

Comment:
Should it be "CONTROLLA LE INTERSEZIONI"? Check intersections in Italian sounds better... Also determiners are generally need in Italian